code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -235,10 +236,24 @@
 			if(heat_level_3 to INFINITY)
 				H.apply_damage(HEAT_GAS_DAMAGE_LEVEL_3, BURN, "head", used_weapon = "Excessive Heat")
 				H.fire_alert = max(H.fire_alert, 2)
+	return 1
 
-	if(!istype(H.wear_suit, /obj/item/clothing/suit/space/eva/plasmaman) || !istype(H.head, /obj/item/clothing/head/helmet/space/eva/plasmaman))
-		to_chat(H, "<span class='warning'>Your body reacts with the atmosphere and bursts into flame!</span>")
-		H.adjust_fire_stacks(0.5)
-		H.IgniteMob()
+/datum/species/plasmaman/handle_life(var/mob/living/carbon/human/H)
```

Comment:
if(environment && environment.total_moles())
         if(environment.oxygen && environment.oxygen >= OXYCONCEN_PLASMEN_IGNITION)

Means that total_moles is useless. The point was to have them ignite at 1% oxygen concentration